R/caret_tidiers.R

```diff
@@ -0,0 +1,111 @@
+#' Tidying methods for confusionMatrix objects
+#'
+#' Tidies the result of confusion matrix from the \code{caret} package.
+#' Only a \code{tidy} method is provided, not an \code{augment} or
+#' \code{glance} method.
+#'
+#' @param x An object of class \code{confusionMatrix}
+#' @param show_class A boolean of whether to show the values for class specific
+#' quantities from the confusion matrix (specificty, sensitivity, etc.). If set
+#' to FALSE, result will only show accuracy and kappa.
+#' @param ... extra arguments (not used)
+#'
+#' @return A data.frame with one or more of the following columns:
```

Comment:
Should return a tibble!

Reply:
I think I need to fix the durbinWatsonTest as well then.